### ISSUE TITLE
[Quest]Saving Sharpbreak ID  2994

### ISSUE THREAD

**Issue by sarahkay88** [WoWManiaUK/Blackwing-Lair#352]
Quest:https://www.wowhead.com/quest=2994/saving-sharpbeak 
Wow head says they made it obsolete, unsure of when it was taken out or weather this is an issue if not then feel free to close just wanted to double check.
NPC https://www.wowhead.com/npc=43157/doran-steelwing still provided you with the gryphon to fly up and there is also a gryphon to come back down
My enquire is should this quest be available?
![world of warcraft 03_01_2019 19_55_46](https://user-images.githubusercontent.com/42302580/50658537-9a9a7880-0f91-11e9-9c3c-491675f7c240.png)

**Comment by Dranah** [comment]
Yes it should. Is the quest working fine?
PS:For any further similar questions, you can simply scroll down the comments on the wowhead page of the quest and see if you find any from the cata expansion. If you do, and they do not speak about how sorry they are that this quest is gone, then it was still present

**Comment by sarahkay88** [comment]
No. that was my enquiry it doesn't supply the quest just gives the speech n gryphon to go to the area.

**Comment by Dranah** [comment]
So, missing quest.

**Comment by sarahkay88** [comment]
Yup
